### ISSUE TITLE
A FontForge Settings.app for mac 

### ISSUE THREAD

**Issue by larsenwork** [fontforge/fontforge#2320]
I've thought about making a small script/app that would allow the user to do a couple of things quickly on a mac such as:

* Change between dark and bright UI quickly
* Reset shortcuts to default mac
* Switch so ⌘ is always second last - although I thought FF was supposed to do this automatically by now?
* Any other ideas?

My thought being that it's a lot quicker to code this using e.g. applescript or cocoa and keeping platform specific stuff out of the big old monster that is FF.

@pathumego @davelab6

**Comment by davelab6** [comment]
Why not use Python so its done within FontForge?

**Comment by larsenwork** [comment]
Since most of it's platform specific and I don't know python :yum:

**Comment by davelab6** [comment]
Haha okay :) I think this is a good idea, especially as with your installer
disk image then its possible to include this along with the app easily :)

**Comment by larsenwork** [comment]
Yeah, thought about including a short readme.pdf with a screenshot or two too (I think you have a better idea of what instructions people need)

**Comment by pathumego** [comment]
Nice! Some ideas, just spitting out everything without crossing off :)

- add bookmark folders?
- Set ui size? A nice slider maybe
- maybe choose any other downloaded theme.
- install python scripts.
- set up word Lists (this is a new feature I've been thinking about, a
python script to auto load word list files)

Write it, we can ask @lpsandaruwan to implement it in python and for GNU
and Win we can ship it packaged. I'll think about platform specific stuff.

**Comment by larsenwork** [comment]
* bookmarks = shortcuts = favorites :)
* UI size = gDraw screen width in inches    - or something else?
* choose downloaded theme is already possible, it should be renamed but I think we've already talked about it in the doc or somewhere else

Generally all platform agnostic stuff should be kept inside FF if possible and 

The only mac specific thing is maybe
* set bookmark folders/shortcuts to mac default - which we could ask people in the installation process

I will make a separate issue for themes and keyboard shortcuts as I have some ideas

**Comment by larsenwork** [comment]
@lpsandaruwan probably already now this, but if/when you are doing many changes to the menu and UI remember to split it up i many commits/pr's so it's easy for us to see in the code what's changed :+1:

**Comment by pathumego** [comment]
+1

**Comment by larsenwork** [comment]
just so I don't forget for the installer

* [ ] Replace x11 icon with ff: option
* [ ] Bookmarks to mac default: option
* [ ] mac-icons: true

**Comment by pathumego** [comment]
@larsenwork 
- [ ]`Add custom Encodings` Copy a given `Encodings.ps` into `.config/fontforge`

**Comment by larsenwork** [comment]
@pathumego not something I've played around with before - what's it for?

**Comment by pathumego** [comment]
It adds custom encodings to the encoding menu.
![screen shot 2015-06-07 at 6 38 45 pm](https://cloud.githubusercontent.com/assets/6846609/8024082/7ababeba-0d44-11e5-8e0f-ad20d0b68a16.png)

**Comment by larsenwork** [comment]
Ah, but isn't that more of a platform independent function that we'd rather have "inside" fontforge?
